usaspending_api/common/etl/spark.py

```diff
@@ -519,3 +526,19 @@ def create_ref_temp_views(spark: SparkSession):
         """
         spark.sql(spark_sql)
     logger.info(f"Created the reference views in the global_temp database")
+    broker_ref_tables = [rds_ref_table._meta.db_table for rds_ref_table in _BROKER_PROXY_TABLES]
+
+    logger.info(f"Creating the following tables under the global_temp database: {broker_ref_tables}")
+    for broker_rdf_table in broker_ref_tables:
+        spark_sql = f"""
+         CREATE OR REPLACE GLOBAL TEMPORARY VIEW {broker_rdf_table}
+         USING JDBC
+         OPTIONS (
+           driver '{jdbc_conn_props["driver"]}',
+           fetchsize '{jdbc_conn_props["fetchsize"]}',
+           url '{get_usas_jdbc_url()}',
```

Comment:
You are using the USAspending JDBC connection here to reference data that will live on Broker.

Reply:
changed to get_broke_jdbc